### DISCUSSION TITLE
Oracle Linux -/bin/postal: line 20: docker-compose: command not found

### DISCUSSION THREAD

**Discussion by jpcapone** [postalserver/postal#2454] [Help with using Postal]
I am running through the installation of postal on an Oracle Linux box and experiencing the error listed in the title - when running **postal initialize**.  I can confirm that docker compose is installed with `[root@mail bin]# docker compose version
Docker Compose version v2.17.3`  
Any suggestions to resolve this issue would be appreciated.

EDIT: I figured this out.  I was downloading the wrong file type for docker-compose.  after downloading the file with the right architecture now i am getting the error:  `exec /docker-entrypoint.sh: exec format error`  when i run postal initialize.  
Any suggestions?

**Comment by willpower232** [comment]
Unfortunately it sounds like you're using an ARM based server which means you won't be able to run the Postal docker images as they are only provided in x86.
